dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -11,7 +11,8 @@
 # See http://support.applitools.com/customer/en/portal/articles/2099488-match-timeout
 MATCH_TIMEOUT = 5
 
-When(/^I open my eyes to test "([^"]*)"$/) do |test_name|
+# A Feature can optionally specify the stitch mode ('css' or 'scroll') for Eyes to create the full screenshot.
+When(/^I open my eyes to test "([^"]*)"( "([^"]*)")?$/) do |test_name, stitch_mode|
```

Comment:
please add a few more words to the name of this step, so that it reads clearly when used without the `<stitch-mode>` placeholder. e.g. `Then I see no difference for "initial load" using stitch mode "css"`

Reply:
Done!  I was avoiding that because I didn't want to parse the capture for the string `scroll` or `css`.  Let me know if there are alternate ways to do this.

Reply:
ah, sorry to make you do extra work! I don't fully understand the syntax, but for next time it appears we have an example of a non-capturing group here:
https://github.com/code-dot-org/code-dot-org/blob/beb3c88a6cda8513ff90c9381ec32ea9d0c664c6/dashboard/test/ui/features/step_definitions/steps.rb#L1022-L1023

Reply:
Thanks for the tip!  That's much better.  I switched to a non-capturing group.